test/e2e/flux_test.go

```diff
@@ -20,7 +20,6 @@ const (
 func runUpgradeFlowWithFlux(test *framework.ClusterE2ETest, updateVersion v1alpha1.KubernetesVersion, clusterOpts ...framework.ClusterE2ETestOpt) {
 	test.GenerateClusterConfig()
 	test.CreateCluster()
-	test.ValidateFlux()
```

Comment:
Do we some kind of wait tho?  Wont upgradecluster end up failing if the state as expected?

Reply:
Yea I added this check thinking that we probably want to validate that flux was installed after create cluster itself

Reply:
Spoke with @danbudris and because we had this before, this is a quick addressing of the issue while we unblock the e2e tests and follow up with checking for the phase in the `ValidateFlux`

Reply:
once the issue is narrowed down we can follow up with https://github.com/aws/eks-anywhere/pull/1484 which adds the scaling checks to the flux validations